### PR TITLE
Version and no admin styles fixes

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2016-02-24 at 11 43 58|screen shot 2016-02-24 at 11 43 47|
screen shot 2016-02-24 at 13 57 34|screen shot 2016-02-24 at 13 56 39|

### CODE REVIEW

File: filer/private/sass/components/_base.scss [hunk 1]
```diff
@@ -65,13 +76,15 @@ body {
...
+    #container {
+        min-width: auto;
+        #content {
+            margin-top: 90px !important;
+            padding: 0;
+            box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.2);
+        }
     }
-    h1.folder_header {
+        h1.folder_header {
```
Comment (vxsx) [inline filer/private/sass/components/_base.scss]: that's a weird indent

File: filer/private/sass/components/_navigator.scss [hunk 2]
```diff
@@ -6,12 +6,17 @@
...
     border-top: solid 1px $gray-lighter !important;
-    tbody {
-        td {
-            padding: 5px 10px !important;
-        }
+    thead th,
+    tbody td {
+        text-align: left;
+        vertical-align: middle;
+        padding: 5px 10px!important;
```
Comment (dhorelik) [inline filer/private/sass/components/_navigator.scss]: space before `!important`, maybe?

File: filer/private/sass/components/_navigator.scss [hunk 3]
```diff
@@ -6,12 +6,17 @@
...
+        border-left: 0!important;
+        border-bottom: 1px solid $gray-lighter;
+        background: 0 0!important;
```
Comment (dhorelik) [inline filer/private/sass/components/_navigator.scss]: `background: none`?

File: filer/private/sass/components/_base.scss [hunk 4]
```diff
@@ -1,5 +1,16 @@
 //##############################################################################
 // BASE
+*,
+*:before, *:after {
+  box-sizing: inherit !important;
```
Comment (dhorelik) [inline filer/private/sass/components/_base.scss]: maybe use something less specific here? Not sure what this fix is for

Comment (dhorelik) [inline filer/private/sass/components/_base.scss]: needs 2 more spaces as well

File: filer/private/sass/components/_base.scss [hunk 5]
```diff
@@ -1,5 +1,16 @@
...
+}
+.filebrowser #content,
+.popup #content {
```
Comment (dhorelik) [inline filer/private/sass/components/_base.scss]:
```
.filebrowser,
.popup {
    #content {
       ...
    }
}
```

File: filer/private/sass/components/_navigator.scss [hunk 6]
```diff
@@ -6,12 +6,17 @@
...
-    tbody {
-        td {
-            padding: 5px 10px !important;
-        }
+    thead th,
+    tbody td {
+        text-align: left;
+        vertical-align: middle;
+        padding: 5px 10px!important;
+        border-left: 0!important;
```
Comment (vxsx) [inline filer/private/sass/components/_navigator.scss]: space

**Comment by vxsx** [comment]
needs a comment about bootstrap thing, but overall LGTM

**Comment by dhorelik** [comment]
LGTM

**Pull request merged**